library/src/scala/runtime/Invoker.scala

```diff
@@ -0,0 +1,55 @@
+package scala.runtime
```

Comment:
I suggest puting this in a scala.runtime.coverage sub-package since to make it clear this is only used by the coverage mode.

Reply:
done